### PR TITLE
Feature (Leaf) - Add tag

### PR DESCRIPTION
**Pull request by meunomeecris**
## Description, motivation and context
- add `Tag` components with status
-  add `Auxiliary Colors` to the `Leaf/Environment`

## Screenshots and GIFs (Artifacts)
Screenshot 2023-10-19 at 13 11 50

## Checklist :leaves:

Thank you!

### CODE REVIEW

File: mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafTag.swift [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+//  
+//
+//  Created by Cris Messias on 15/10/23.
+//
+
+import SwiftUI
+
+struct LeafTag: View {
+    private let text: String
+    private var status: StatusTag
```
Comment (viniciusaro) [inline mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafTag.swift:12]: Can this be a let?
```suggestion
    private let status: StatusTag
```

File: mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafTag.swift [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        LeafTag("accepted", status: .success)
+        LeafTag("rejected", status: .error)
+        LeafTag("closed", status: .disabled)
+    }
+})
+
+
+///
+enum StatusTag: CaseIterable {
+    case success, error, auxiliar, disabled
```
Comment (viniciusaro) [inline mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafTag.swift:62]: What is an `auxiliar` status?

File: mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafTag.swift [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        LeafTag("open", status: .auxiliar)
+        LeafTag("accepted", status: .success)
+        LeafTag("rejected", status: .error)
+        LeafTag("closed", status: .disabled)
+    }
+})
+
+
+///
+enum StatusTag: CaseIterable {
```
Comment (viniciusaro) [inline mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafTag.swift:61]: Is there a specific reason for conforming to the `CaseIterable` protocol?

File: mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafTag.swift [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+#Preview(body: {
+    VStack{
+        LeafTag("open", status: .auxiliar)
+        LeafTag("accepted", status: .success)
+        LeafTag("rejected", status: .error)
+        LeafTag("closed", status: .disabled)
+    }
+})
```
Comment (viniciusaro) [inline mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafTag.swift:57]: Not that important, but I'd rather keep #Preview s at the end of the file, what do you think?

File: mobile/evidence/ios/Leaf/Sources/Leaf/Theme/LeafEnvironment.swift [hunk 5]
```diff
@@ -51,6 +57,12 @@ public let darkLeafTheme = LeafTheme(
             secondary: .gray,
             tertiary: .gray,
             quaternary: .green
+        ),
+        auxiliar: LeafTheme.AuxiliarColor(
+            auxiliar: Color(red:74/255, green: 179/255, blue: 255/255),
+            success:  Color(red:68/255, green: 177/255, blue: 16/255),
+            error: Color(red:249/255, green: 100/255, blue: 90/255),
+            disabled: Color(red:162/255, green: 168/255, blue: 159/255)
```
Comment (viniciusaro) [inline mobile/evidence/ios/Leaf/Sources/Leaf/Theme/LeafEnvironment.swift:65]: I think `accepted`, `rejected`, `open` are important concepts in the specific domain of our application. Maybe we should have color tokens for those concepts?

**Review by viniciusaro — COMMENTED** [review]
PRETTY GOOD MAFREND! 😍

File: mobile/evidence/ios/Leaf/Sources/Leaf/Theme/LeafEnvironment.swift (hunk 5 above)
Comment (meunomeecris) [reply mobile/evidence/ios/Leaf/Sources/Leaf/Theme/LeafEnvironment.swift:65]: So,  those colors are exactly for those concepts.  Auxiliary: open, success: accepted, error: rejected, and disabled: closed. I will change the names. I created like this bc I thoughts reuse like error, success, etc.

Comment (meunomeecris) [reply mobile/evidence/ios/Leaf/Sources/Leaf/Theme/LeafEnvironment.swift:65]:
```swift

``` tag: LeafTheme.TagColor(
            open: Color(red:3/255, green: 129/255, blue: 220/255),
            accepted: .green,
            rejected: Color(red:249/255, green: 100/255, blue: 90/255),
            closed: Color(red:121/255, green: 128/255, blue: 117/255)
        )

Comment (meunomeecris) [reply mobile/evidence/ios/Leaf/Sources/Leaf/Theme/LeafEnvironment.swift:65]: Eu fiz essas alterações :D

File: mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafTag.swift (hunk 4 above)
Comment (meunomeecris) [reply mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafTag.swift:57]: YES CHEFE! hahah

File: mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafTag.swift (hunk 2 above)
Comment (meunomeecris) [reply mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafTag.swift:62]: good question haha

**Comment by meunomeecris** [comment]
Changes done!

**Review by viniciusaro — APPROVED** [review]

**Pull request merged**